### DISCUSSION TITLE
Demo Video not working in Firefox

### DISCUSSION THREAD

**Discussion by NyCodeGHG** [utilyre/barbecue.nvim#58] [General]
The demo video seems to use YUV444 chroma subsampling which is not supported by firefox. It would be great to have the demo video supported in firefox.

**Comment by utilyre** [comment]
I have no idea about video formats and their web support, can you make a PR? (the uploaded video is the original recording)

**Reply by NyCodeGHG** [reply]
yes, i can make a PR. Should be pretty easy to fix this using ffmpeg
